embedding_transfer_learning/steps.py

```diff
@@ -28,86 +28,86 @@ def do(self, previous_states: Dict):
         Args:
             previous_states (Dict): previous states
         """
-        art_logger.info("Reading data")
+        if QUERIES_PATH.exists() and RECIPIES_PATH.exists():
```

Comment:
Actually the hash checking was taking care of that, why is it here?

Reply:
If someone downloads data one doesn't have to go through preprocessing step. This step may not even run or take very long time if run on CPU